scripts/testnet/sovereignBridge/pyScripts/update_toml.py

```diff
@@ -18,26 +18,69 @@ def update_subscribed_addresses(lines, section, identifier, main_chain_address)
     return updated_lines
 
 
+def update_key(lines, key, value) -> []:
+    updated_lines = []
+
+    for line in lines:
+        if key in line:
+            line = re.sub(rf'{re.escape(key)}\s*=\s*".*?"', f'{key} = "{value}"', line)
+        updated_lines.append(line)
+
+    return updated_lines
+
+
+def update_sovereign_config(file_path, main_chain_address, sovereign_chain_address):
+    with open(file_path, 'r') as file:
+        lines = file.readlines()
+
+    updated_lines = update_subscribed_addresses(lines, "OutgoingSubscribedEvents", "deposit", sovereign_chain_address)
+    updated_lines = update_subscribed_addresses(updated_lines, "NotifierConfig", "deposit", main_chain_address)
+    updated_lines = update_subscribed_addresses(updated_lines, "NotifierConfig", "execute", main_chain_address)
+
+    with open(file_path, 'w') as file:
+        file.writelines(updated_lines)
+
+
+def update_esdt_prefix(file_path, esdt_prefix):
+    with open(file_path, 'r') as file:
+        lines = file.readlines()
+
+    updated_lines = update_key(lines, "ESDTPrefix", esdt_prefix)
+
+    with open(file_path, 'w') as file:
+        file.writelines(updated_lines)
+
+
+def update_transfer_and_execute_address(file_path, address):
+    with open(file_path, 'r') as file:
+        lines = file.readlines()
+
+    for i, line in enumerate(lines):
+        if "TransferAndExecuteByUserAddresses" in line:
+            lines[i + 1] = re.sub(r'".*?"', f'"{address}"', lines[i + 1])
+
+    with open(file_path, 'w') as file:
+        file.writelines(lines)
+
+
 def main():
     # input arguments
     main_chain_address = sys.argv[1]
     sovereign_chain_address = sys.argv[2]
+    esdt_prefix = sys.argv[3]
 
     current_path = os.getcwd()
     project = 'mx-chain-go'
     index = current_path.find(project)
     project_path = current_path[:index + len(project)]
     toml_path = project_path + "/cmd/sovereignnode/config/sovereignConfig.toml"
+    update_sovereign_config(toml_path, main_chain_address, sovereign_chain_address)
 
-    with open(toml_path, 'r') as file:
-        lines = file.readlines()
-
-    updated_lines = update_subscribed_addresses(lines, "OutgoingSubscribedEvents", "deposit", sovereign_chain_address)
-    updated_lines = update_subscribed_addresses(updated_lines, "NotifierConfig", "deposit", main_chain_address)
-    updated_lines = update_subscribed_addresses(updated_lines, "NotifierConfig", "execute", main_chain_address)
+    toml_path = project_path + "/cmd/node/config/systemSmartContractsConfig.toml"
+    update_esdt_prefix(toml_path, esdt_prefix)
 
-    with open(toml_path, 'w') as file:
-        file.writelines(updated_lines)
+    toml_path = project_path + "/cmd/node/config/config.toml"
+    update_transfer_and_execute_address(toml_path, sovereign_chain_address)
```

Comment:
[totally optional]
Can't we include `update_transfer_and_execute_address` & `update_esdt_prefix` in a single `updateConfigs`, which calls: -> `update_sovereign_config` + `update_node_config` (new func for `cmd/node/config`)